bytestring.cabal

```diff
@@ -165,6 +165,15 @@ test-suite test-builder
   ghc-options:      -Wall -fwarn-tabs -threaded -rtsopts
   default-language: Haskell2010
 
+test-suite bytestring-th
```

Comment:
Please put this into one of existing test suites. Linking an additional executable is expensive on Windows.

Reply:
In which of three it belongs? lazy-hclose and test-builder seems to be testing specific features, and prop-compiled is for properties. It looks like fourth test-suite for an isolated feature is following the example.